src/model.js

```diff
@@ -2571,7 +2571,13 @@ Specify a different name for either index to resolve this issue.`);
       }
     }
 
-    const instances = records.map(values => this.build(values, { isNewRecord: true, include: options.include }));
+    const instances = records.map(values => this.build(values, {
+      isNewRecord: true,
+      attributes: options.fields,
```

Comment:
Can you add a test for `attributes`?

Reply:
Sorry, attributes is not a valid missing option, I notice the test for attributes is already passed in integration test. You can check the integration test for bulk-create.

Reply:
Sorry, `attributes` is not a valid missing option, I notice the test for attributes is already passed in integration test. You can check the integration test for bulk-create.